esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -3103,8 +3105,11 @@ void CFileSprayEx::addDropZoneFile(IEspContext& context, IDirectoryIterator* di,
     files.append(*aFile.getLink());
 }
 
-void CFileSprayEx::searchDropZoneFiles(IEspContext& context, const char* server, const char* dir, const char* nameFilter, IArrayOf<IEspPhysicalFileStruct>& files, unsigned& filesFound)
+void CFileSprayEx::searchDropZoneFiles(IEspContext& context, const char* dropZoneName, const char* server, const char* dir, const char* nameFilter, IArrayOf<IEspPhysicalFileStruct>& files, unsigned& filesFound)
 {
+    if (getDropZoneScopePermissions(context, dropZoneName, server, dir) < SecAccess_Read)
```

Comment:
I believe this only requires SecAccess_Access ?

Reply:
Same as the line 2976